packages/js/src/feeds/notification.ts

```diff
@@ -35,20 +35,20 @@ export class Notification implements Pick<NovuEventEmitter, 'on' | 'off'> {
   #emitter: NovuEventEmitter;
   #apiService: ApiService;
 
-  _id: string;
-  _feedId?: string | null;
-  createdAt: string;
-  updatedAt: string;
-  actor?: Actor;
-  subscriber?: Subscriber;
-  transactionId: string;
-  content: string;
-  read: boolean;
-  seen: boolean;
-  deleted: boolean;
-  cta: Cta;
-  payload: Record<string, unknown>;
-  overrides: Record<string, unknown>;
+  readonly _id: string;
```

Comment:
Let's remove all the fields we don't need on the FE from this Class for now and we will clean them from the API at a later stage.